docs/6lowpan-ble.md

```diff
@@ -14,23 +14,29 @@ should work with a 4.0 compliant BT/BLE adapter.
 Please follow the ZJS initial setup in the main [README](../README.md)
 
 ## Building for A101
-Building the OCF server sample is the same as any other:
+Building any IP networking sample is the same as any other, except there are
+two optional variables that can be used to set the BLE MAC address as well as
+the device name, `BLE_ADDR` and `DEVICE_NAME`. Here are some examples of
+building various samples:
 ```bash
-$ make BOARD=arduino_101 JS=samples/OcfServer.js
+$ make BOARD=arduino_101 JS=samples/OcfServer.js BLE_ADDR="DE:AD:BE:EF:11:22"
+$ make BOARD=arduino_101 JS=samples/TCPEchoServ6.js DEVICE_NAME="my_tcp_server"
 ```
+When doing BLE discovery (explained below) you should see your device name and
+MAC address appear if you use these options.
 
-## Flashing the A101
-Flash the Arduino 101 as you would any other script:
-```bash
-$ make dfu
-```
```

Comment:
I figured the user should know how to flash, this doc was really just meant to describe the BLE connection procedure and different make variables. I can put it back in if you want though